src/common/DisabledButtonInfo.tsx

```diff
@@ -0,0 +1,34 @@
+import React, { ReactNode, useEffect } from 'react'
+import Tooltip, { TooltipProps } from '@mui/material/Tooltip'
+
+function DisabledButtonInfo({
+  conditions,
+  placement,
+  children,
+}: {
+  conditions: string[]
+  placement?: TooltipProps['placement']
```

Comment:
Personal preference is to place optional properties at the bottom (with any type definition). I find it helps when quickly scanning code, but not something worth changing unless it's a convention we want to enforce

Reply:
I like this to be honest, perhaps we should do this from now on.